cifar10/utils.py

```diff
@@ -170,6 +169,19 @@ def vit_b4_32x32(num_classes=10, input_channels=3):
         attn_drop_rate=0.0,
     )
 
+def vit_b3_224x244(num_classes=10, input_channels=3):
```

Comment:
Why b3 ? It is b16 ?

Reply:
Could you elaborate on that ?

Reply:
It's name convention they have: bX means base with patch of size X.
Maybe, we should have the same conventions as in the paper or other code sources...

Reply:
Agreed let's write b16 ! :p